src/components/lyne-card-product/lyne-card-product.stories.js

```diff
@@ -21,6 +21,24 @@ const wrapperStyle = (context) => {
   return `background-color: ${ColorMilkDefault};`;
 };
 
+/* ************************************************* */
+/* Documentation platform container                  */
```

Comment:
I guess you can remove this comment block?

Reply:
I would like to introduce this change in a subsequent commit/merge with master. For this reason it is commented out.